src/t8_forest/t8_forest_iterate.h

```diff
@@ -44,20 +44,30 @@ typedef int (*t8_forest_iterate_face_fn) (t8_forest_t forest, t8_locidx_t ltreei
  * \param[in] leaf_elements       the leaf elements in \a forest that are descendants of \a element (or the element 
  *                                itself if \a is_leaf is true)
  * \param[in] tree_leaf_index     the local index of the first leaf in \a leaf_elements
- * \param[in] queries             if not NULL, a query that is passed through from the search function
```

Comment:
The element parameter comment says "the element for which the query is executed" which should be corrected.

Reply:
Thanks for the reminder! I updated the comment.